specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -733,6 +789,12 @@
         },
         "trainResult": {
           "$ref": "#/definitions/TrainResult"
+        },
+        "composedTrainedResults": {
```

Comment:
>composedTrainedResults [](start = 9, length = 22)

this should be dictionary of id -> training results rather than just TrainResult array.